lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidumpYaml.py

```diff
@@ -0,0 +1,142 @@
+"""
+Test saving a mini dump, from yamilized examples.
+"""
+
+import os
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
```

Comment:
Maybe we should test some degenerate cases like empty ranges and ranges that span exactly the full memory region?

Reply:
There is actually `TestProcessSaveCoreMinidump` which covers these cases! I covered the new intersections cases but we have a test for invalid regions, empty regions and full ones.